src/action/linux/start_systemd_unit.rs

```diff
@@ -90,9 +90,13 @@ impl Action for StartSystemdUnit {
         tracing::debug!("Stopping systemd unit");
 
         // TODO(@Hoverbear): Handle proxy vars
-        execute_command(Command::new("systemctl").arg("stop").arg(format!("{unit}")))
-            .await
-            .map_err(|e| StartSystemdUnitError::Command(e).boxed())?;
+        execute_command(
+            Command::new("systemctl")
+                .arg("disable")
```

Comment:
Is `disable` more forceful than `stop`? Or is this just so it doesn't get started again (by e.g. the socket)?

Reply:
The `systemctl` docs state `disable` is the reverse: https://www.freedesktop.org/software/systemd/man/systemctl.html :woman_shrugging: